### PR TITLE
Fix/knt1 896 aio sdk demo feedback fixes

### PR DESCRIPTION
**Pull request by kafkaphoenix**
## Description

- Add channel to send error in messaging module
- Make description optional in the method register model 
- Return error in register model if the model already exists
- Move scope from messaging module in go to centralized config
- Return not found error in predictions python update method

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other changes (ci configuration, documentation or any other kind of changes)

## Checklist:

- [ ] I have created tests for my code changes, and the tests are passing.
- [ ] I have executed the pre-commit hooks locally.
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: go-sdk/sdk/centralized-configuration/centralized_configuration_delete_test.go [hunk 1]
```diff
@@ -3,16 +3,15 @@
 package centralizedconfiguration_test
 
 import (
-	centralizedConfiguration "github.com/konstellation-io/kai-sdk/go-sdk/sdk/centralized-configuration"
-	"github.com/konstellation-io/kai-sdk/go-sdk/sdk/messaging"
+	centralizedconfiguration "github.com/konstellation-io/kai-sdk/go-sdk/sdk/centralized-configuration"
```
Comment (dfinteligenz) [inline go-sdk/sdk/centralized-configuration/centralized_configuration_delete_test.go:6]:
```suggestion
	centralizedConfiguration "github.com/konstellation-io/kai-sdk/go-sdk/sdk/centralized-configuration"
```

Use camelCase for Go.

File: go-sdk/sdk/centralized-configuration/centralized_configuration_get_test.go [hunk 2]
```diff
@@ -8,7 +8,6 @@ import (
 
 	"github.com/konstellation-io/kai-sdk/go-sdk/mocks"
 	centralizedconfiguration "github.com/konstellation-io/kai-sdk/go-sdk/sdk/centralized-configuration"
```
Comment (dfinteligenz) [inline go-sdk/sdk/centralized-configuration/centralized_configuration_get_test.go:10]:
```suggestion
	centralizedConfiguration "github.com/konstellation-io/kai-sdk/go-sdk/sdk/centralized-configuration"
```
Ditto, replace with camelCase

File: go-sdk/sdk/centralized-configuration/centralized_configuration_test.go [hunk 3]
```diff
@@ -8,15 +8,14 @@ import (
 
 	"github.com/konstellation-io/kai-sdk/go-sdk/internal/common"
 
-	centralizedConfiguration "github.com/konstellation-io/kai-sdk/go-sdk/sdk/centralized-configuration"
+	centralizedconfiguration "github.com/konstellation-io/kai-sdk/go-sdk/sdk/centralized-configuration"
```
Comment (dfinteligenz) [inline go-sdk/sdk/centralized-configuration/centralized_configuration_test.go:11]: Ditto
```suggestion
	centralizedConfiguration "github.com/konstellation-io/kai-sdk/go-sdk/sdk/centralized-configuration"
```

File: go-sdk/sdk/model-registry/model_registry.go [hunk 4]
```diff
@@ -72,13 +72,17 @@ func New(logger logr.Logger, meta *metadata.Metadata) (*ModelRegistry, error) {
...
 	ctx := context.Background()
 
 	if name == "" {
 		return errors.ErrEmptyName
 	}
 
+	if len(description) == 0 {
+		description = []string{""}
+	}
+
```
Comment (dfinteligenz) [inline go-sdk/sdk/model-registry/model_registry.go:85]: I'd replace this code with a local `modelDescription` variable initialized as an empty string.

And if the `len(description) > 0`, then set `description = description[0]`

**Review by dfinteligenz — CHANGES_REQUESTED** [review]

File: go-sdk/sdk/centralized-configuration/centralized_configuration.go [hunk 5]
```diff
@@ -20,6 +19,16 @@ const (
...
 )
 
+type Scope string
+
+const (
+	GlobalScope          Scope = "global"
+	ProductScope         Scope = "product"
+	WorkflowScope        Scope = "workflow"
+	ProcessScope         Scope = "process"
+	_messagingLoggerName       = "[centralizedconfiguration]"
```
[the remark this answers is not in the document]
Comment (kafkaphoenix) [reply go-sdk/sdk/centralized-configuration/centralized_configuration.go]: oh right i forgot this one thanks

**Review by dfinteligenz — APPROVED** [review]

**Pull request merged**